tests/test_errata_tool_product.py

```diff
@@ -150,3 +151,51 @@ def test_basic(self, client):
             'exd_org_group': 'Cloud',
         }
         assert product == expected
+
+
+class TestPrepareDiffData(object):
+
+    def test_diff_from_none(self):
+        after_data = {
+            'short_name': 'RHNEW',
+            'description': 'new thing',
+        }
+
+        expected_output = {
+            'before': {},
+            'after': after_data,
+            'before_header': "Not present",
+            'after_header': "New product 'RHNEW'",
+        }
+
+        assert expected_output == prepare_diff_data(None, after_data)
```

Comment:
This is really cosmetic, but I think it's a lot easier to read naturally that `<result> is <expected>` instead of `<expected> is <result`>`

```suggestion
        assert prepare_diff_data(None, after_data) == expected_output
```

Reply:
Agreed. :+1: